demonstrations/tutorial_rosalin.py

```diff
@@ -561,7 +561,10 @@ def step(self, params):
     params = opt.step(params)
     cost_rosalin.append(cost_analytic(params))
     shots_rosalin.append(opt.shots_used)
-    print("Step {}: cost = {} shots_used = {}".format(i, cost_rosalin[-1], shots_rosalin[-1]))
+    print(
+        "Step {}: cost = {}, shots_used = {}".format(i, cost_rosalin[-1], shots_rosalin[-1])
+
+    )

```

Comment:
GitHub added in an extra line!

Although, we could use f-strings to make the line shorter.
```suggestion
    print(f"Step {i}: cost = {cost_rosalin[-1]}, shots_used = {shots_rosalin[-1]}")
```